cli/lib/exec/spawn.js

```diff
@@ -140,6 +140,13 @@ module.exports = {
         function resolveOn (event) {
           return function (code, signal) {
             debug('child event fired %o', { event, code, signal })
+
+            if (code === null && signal) {
```

Comment:
the node.js docs state:

>  If the process exited, code is the final exit code of the process, otherwise null. If the process terminated due to receipt of a signal, signal is the string name of the signal, otherwise null. One of the two will always be non-null.

so this should just be:

```suggestion
            if (code === null) {
```

https://devdocs.io/node/child_process#child_process_event_exit

Reply:
hehe, this was more to be precise especially when doing testing to avoid sending `code: null, signal: null` via stub